admin/src/scenes/dashboard/missions/status.js

```diff
@@ -1,28 +1,43 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import { Col, Row } from "reactstrap";
 import { Link } from "react-router-dom";
-import { CardArrow, Card, CardTitle, CardValueWrapper, CardValue, CardPercentage } from "../../../components/dashboard";
+import { CardArrow, Card, CardTitle, CardValueWrapper, CardValue, CardPercentage, CardSubtitle } from "../../../components/dashboard";
 import { translate, MISSION_STATUS, MISSION_STATUS_COLORS } from "../../../utils";
 
 export default function Status({ data, filter, getLink }) {
-  const total = Object.keys(data).reduce((acc, a) => acc + data[a], 0);
+  const [total, setTotal] = useState();
+
+  useEffect(() => {
+    if (Object.keys(data).length !== 0) {
+      console.log("here", data);
+      setTotal(Object.keys(data).reduce((acc, a) => acc + data[a].doc_count, 0));
+    }
+  }, [data]);
+
+  if (Object.keys(data).length === 0) {
+    return null;
+  }
+
   return (
     <React.Fragment>
       <h3 className="mt-4 mb-2 text-xl">Statut des missions proposées par les structures</h3>
       <Row>
         {Object.values(MISSION_STATUS).map((l, k) => {
+          if (!data[l]) return null;
           return (
             <Col md={6} xl={3} key={k}>
               <Link to={getLink({ base: `/mission`, filter, filtersUrl: [`STATUS=%5B"${l}"%5D`] })}>
                 <Card borderBottomColor={MISSION_STATUS_COLORS[l]}>
-                  <CardTitle>{translate(l)}</CardTitle>
+                  <CardTitle>{translate(data[l].name)}</CardTitle>
```

Comment:
pourquoi se compliquer la vie ?
```suggestion
                  <CardTitle>{translate(l)}</CardTitle>
```